openapi/v2/core-command.yaml

```diff
@@ -144,7 +145,12 @@ components:
         type: string
       title: Setting
       type: object
-      example: { "AHU-TargetTemperature": "28.5", "AHU-TargetBand": "4.0" }
+      example:
+        AHU-TargetTemperature: "28.5"
```

Comment:
This should still be JSON, correct?

Reply:
yes, in swagger file, we just use yaml to represent the object structure.  it will be JSON in the service response.